sw/device/lib/crt/crt.S

```diff
@@ -132,3 +132,33 @@ L_copy_error:
 
   // Set function size to allow disassembly.
   .size crt_section_copy, .-crt_section_copy
+
+/**
+ * Set the global pointer register (gp) to the value of __global_pointer.
```

Comment:
I think you dropped a trailing `$` here.

Reply:
Done